apps/freelance/src/redux/auth/auth-slice.ts

```diff
@@ -0,0 +1,41 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+import { RootState } from '../store';
+export interface AuthState {
+  name: string | null;
+  token: string | null;
+}
+
+const initialState: AuthState = {
+  name: null,
+  token: null,
+};
+
+const authSlice = createSlice({
+  name: 'user',
+  initialState,
+  reducers: {
+    setUser(state, action: PayloadAction<{ name: string; token: string }>) {
+      localStorage.setItem(
```

Comment:
no, there is redux persist library which will handle it instead of you

Reply:
persist library added